commercial/public/ads.txt

```diff
@@ -19,3 +18,6 @@ teads.tv, 8260, DIRECT, 15a9c44f6d26cbe1
 teads.tv, 8095, DIRECT, 15a9c44f6d26cbe1
 google.com, pub-3746578658400510, DIRECT, f08c47fec0942fa0
 synapsys.us, 174167, DIRECT
+teads.tv, 7607, DIRECT
```

Comment:
Worth querying with Drew about the fact the other teads entries have a further identifier above? Is it of any relevant to us?

Reply:
Yes, I corrected it just afterwards. You were just very fast :)

Reply:
The last part is optional, confirmed with Pascal